src/components/AppRouter.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+import { Switch, Redirect, HashRouter } from 'react-router-dom'
+
+import Home from 'src/components/Home/Home'
+import Onboarding from 'src/components/Onboarding/Onboarding'
+import PapersListWrapper from 'src/components/Papers/PapersListWrapper'
+import FilesViewerWithQuery from 'src/components/Viewer/FileViewerWithQuery'
+import OnboardedGuardedRoute from 'src/components/OnboardedGuardedRoute'
+
+const routes = [
+  {
+    path: '/',
+    component: Home
+  },
+  {
+    path: '/files/:fileCategory',
+    component: PapersListWrapper
+  },
+  {
+    path: '/file/:fileId',
+    component: FilesViewerWithQuery
+  },
+  {
+    path: '/onboarding',
+    component: Onboarding
+  }
+]
+
+export const AppRouter = () => {
```

Comment:
it's time to test AppRouter (you can remove one assertion) :  

```
import React from 'react'
import { render } from '@testing-library/react'
import { AppRouter } from 'src/components/AppRouter'

/* eslint-disable react/display-name */
jest.mock('src/components/OnboardedGuardedRoute', () => ({ component }) => (
  <div data-testid="OnboardedGuardedRoute">{component()}</div>
))
jest.mock('src/components/Home/Home', () => () => <div data-testid="Home" />)
jest.mock('src/components/Onboarding/Onboarding', () => () => (
  <div data-testid="Onboarding" />
))
jest.mock('src/components/Papers/PapersListWrapper', () => () => (
  <div data-testid="PapersListWrapper" />
))
jest.mock('src/components/Viewer/FileViewerWithQuery', () => () => (
  <div data-testid="FileViewerWithQuery" />
))
jest.mock('react-router-dom', () => ({
  HashRouter: ({ children }) => <div data-testid="HashRouter">{children}</div>,
  Redirect: ({ children }) => <div data-testid="Redirect">{children}</div>,
  Switch: ({ children }) => <div data-testid="Switch">{children}</div>
}))
/* eslint-enable react/display-name */

describe('AppRouter', () => {
  it('should render home, onboarding, papersListWrapper, fileViewerWithQuery', () => {
    // When
    const { container, queryByTestId } = render(<AppRouter />)

    // Then
    expect(queryByTestId('Home')).toBeTruthy()
    expect(queryByTestId('Onboarding')).toBeTruthy()
    expect(queryByTestId('PapersListWrapper')).toBeTruthy()
    expect(queryByTestId('FileViewerWithQuery')).toBeTruthy()
    expect(container).toMatchInlineSnapshot(`
      <div>
        <div
          data-testid="HashRouter"
        >
          <div
            data-testid="Switch"
          >
            <div
              data-testid="OnboardedGuardedRoute"
            >
              <div
                data-testid="Home"
              />
            </div>
            <div
              data-testid="OnboardedGuardedRoute"
            >
              <div
                data-testid="PapersListWrapper"
              />
            </div>
            <div
              data-testid="OnboardedGuardedRoute"
            >
              <div
                data-testid="FileViewerWithQuery"
              />
            </div>
            <div
              data-testid="OnboardedGuardedRoute"
            >
              <div
                data-testid="Onboarding"
              />
            </div>
            <div
              data-testid="Redirect"
            />
          </div>
        </div>
      </div>
    `)
  })
})
```

Reply:
Thanks for this nice example 👍